app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    OutputDocumentMailer.guidance_record(appointment_summary).deliver_later
```

Comment:
Firstly - why are we passing in `appointment_summary` not `output_document`?

Secondly - I still think this is a bit confused. 'Output document' and 'Guidance record' are the same thing - we should settle on one.

The specific email is saying 'here is your record for the first time'. I can envisage this may grown in the future to include further emails for things like 'here is your re-issued record of guidance'.

Maybe:

```rb
OutputDocumentMailer.issue(output_document).deliver_later

# or 

OutputDocumentMailer.first_issue(output_document).deliver_later

# or 

OutputDocumentMailer.initial_issue(output_document).deliver_later
```